src/Adapter/Order/CommandHandler/DeleteProductFromOrderHandler.php

```diff
@@ -116,29 +127,6 @@ public function handle(DeleteProductFromOrderCommand $command)
         $this->contextStateManager->restoreContext();
     }
 
-    /**
-     * @param OrderDetail $orderDetail
-     * @param Cart $cart
-     *
-     * @return bool
-     */
-    private function updateCart(OrderDetail $orderDetail, Cart &$cart)
-    {
-        // Update Product Quantity in the cart
-        $cart->updateQty(
-            $orderDetail->product_quantity,
-            $orderDetail->product_id,
-            $orderDetail->product_attribute_id,
-            false,
-            'down'
-        );
-        // Remove & Add CartRules applied to cart
-        CartRule::autoRemoveFromCart();
-        CartRule::autoAddToCart();
-
-        return $cart->update();
-    }
-
```

Comment:
I remove the `updateCart` method, which is the only thing it had in common with the service.